opm/simulators/wells/MultisegmentWell.hpp

```diff
@@ -234,8 +234,12 @@ namespace Opm
         // two off-diagonal matrices
         mutable OffDiagMatWell duneB_;
         mutable OffDiagMatWell duneC_;
-        // diagonal matrix for the well
+        // "diagonal" matrix for the well. It has offdiagonal entries for inlets and outlets.
```

Comment:
Just a comment, not against the change in comment here.

Diagonal here basically means the location of this well matrix in the global matrix, which contains the derivatives of the equations of a well with respect to its own primary variables. 

There are another two matrices associated with this well, which describe the relationship between the well and reservoir. The are on the off diagonal position in the global matrix.

Reply:
Understood